Examples/ElizaSwiftPackageApp/ElizaSwiftPackageApp/Info.plist

```diff
@@ -1,13 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
-<dict>
```

Comment:
Xcode automatically made these changes when it rewrote the file after changing the bundle ID